src/scripts/main.js

```diff
@@ -1,3 +1,9 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+for (let i = 0; i < table.rows.length; i++) {
+  const clone = table.rows[i].cells[1].cloneNode(true);
+
+  table.rows[i].cells[4].before(clone);
```

Comment:
```suggestion
  table.rows[i].lastElementChild.before(clone);
```
Такий варіант підійшов би краще, якби ми не знали точної кількості стовбчиків